app/scripts/views/settings/emails.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
   const EMAIL_REFRESH_DELAYMS = 350;
 
+  const EMAIL_ENABLED_REGEX = /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/;
```

Comment:
Point taken, though I'm not sure the serialized config is the place to put these either. Serialized config is used for things that differ between environments or that change often. The email list doesn't seem like it's either. 

To quickly get this fix into train-87 and deployed, I'm not adverse to leaving the code here and filing a follow on issue so we can bikeshed elsewhere.